Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: install_phoenix create_project apply_template
+
+install_phoenix:
+	printf "Y\n" | mix archive.install hex phx_new ${PHOENIX_VERSION}
+
+create_project:
+	printf "Y\nY\n" | mix phx.new ${VARIANT}_project
+
+apply_template:
+	cd ${VARIANT}_project && \
+	echo '{:nimble_phx_gen_template, path: "../"},' > nimble_phx_gen_template.txt && \
+	sed -i -e '/{:phoenix, "~> /r nimble_phx_gen_template.txt' mix.exs && \
+	rm nimble_phx_gen_template.txt && \
+	mix deps.get && \
+	mix format && \
+	printf "Y\nY\nY\nY\n" | mix nimble.phx.gen.template --${VARIANT}
```

Comment:
The 4 times `Y` is to accept the prompts, right? If there was more options, this build script would not work.

I found an alternative and we should be able to do:

```shell
yes Y | mix nimble.phx.gen.template --${VARIANT}
```

Check it out: https://en.wikipedia.org/wiki/Yes_(Unix)

Reply:
I tried the `yes command` already, that doesn't work @olivierobert as the Yes command will shutdown the shell process and the [Github job will stop](https://github.com/nimblehq/elixir-templates/runs/1181033071?check_suite_focus=true#step:10:57) at that moment.

https://stackoverflow.com/questions/20573282/hudson-yes-standard-output-broken-pipe

Reply:
Noted. Too bad then.

Reply:
Let's document what each `Y` corresponds to. Same for `create_project` as personally I don't recall what they are for. And your future self will not remember either in 6 months 😆

Reply:
Added in https://github.com/nimblehq/elixir-templates/pull/24/commits/96e3f4d93baf339cc65c5a62fbd090729752a4fb @olivierobert